ui/src/pages/users.vue

```diff
@@ -468,6 +582,47 @@ function createUser() {
   }
 }
 
+function hardDeleteUser() {
+  if (usernameConfirmation.value !== editedUser.value.username) {
+    toast.error('Username does not match. Entered: "${usernameConfirmation.value}", Expected: "${editedUser.value.username}". ');
```

Comment:
Instead of showing this error via toast, show use `<va-input>`'s props to show error directly beneath the input. See examples of how to achieve this here: https://ui.vuestic.dev/ui-elements/input#validation-and-error-success-state